packages/components/psammead-image/README.md

```diff
@@ -0,0 +1,39 @@
+# psammead-image &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-image.svg)](https://www.npmjs.com/package/@bbc/psammead-image) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+The `psammead-image` component is a styled `img` element.
+
+## Usage
+
+```jsx
+import Image from '@bbc/psammead-image';
+
+const Wrapper = (src, alt, width, height) => (
+  <Image src={src} alt={alt} width={width} height={height} />
+);
+```
+
+| Prop     | Type          |
+|:---------|:--------------|
+| `alt`    | string        |
+| `height` | number/string |
+| `src`    | string        |
+| `width`  | number/string |
+
+## Accessibility notes
+
+This component requires an alt property to describe the image. This alt text is crucial for users of Assistive Technology, and by any user whose internet connection is so slow the browser decides to render the text instead of the image.
```

Comment:
```suggestion
 This component requires an `alt` property to describe the image. This `alt` text is crucial for users of Assistive Technology, and by any user whose internet connection is so slow the browser decides to render the text instead of the image.
```

Reply:
Without the preceding space:

```suggestion
This component requires an `alt` property to describe the image. This `alt` text is crucial for users of Assistive Technology, and by any user whose internet connection is so slow the browser decides to render the text instead of the image.
```